src/main/webapp/js/user-page-loader.js

```diff
@@ -117,10 +117,17 @@ function fetchAboutMe(){
   })
 }
 
+function addRichTextEditor(){
```

Comment:
Just add a quick javadoc here and I think you're good!

Reply:
Okay, so what's a javadoc? Internet is saying it's a documentation generator, but are you just saying I should comment on it?

Reply:
Yeah, and maybe @acolwell can weight in on if it's absolutely necessary for every function. I found this to describe what I thought was a javadoc, but I'm just learning to used javadoc as well: http://www.drjava.org/docs/user/ch10.html

Reply:
Thanks @drewbern for suggesting the addition of Javadocs. It is a good habit to get into and helps other engineers understand what the function is supposed to do. Even if we aren't generating the documentation with the javadoc tool it still provides a good structure and rules for how to document new functions. In this particular case though we are in javascript code so technically javadoc doesn't apply. I am all for a adding comments to document behavior though. We just don't need to follow the javadoc '/** */' convention here. '/* */' style comments are fine.

@nbaldy I've just got a couple minor comments about your comments here and above.
1. This is javascript so you can just use '/*   */' style. If this was java you were doing a javadoc comment you'd want to end with '*/' and not '**/' like you have here. Not a big deal, just letting you know.
2. The "Removes image plugin..." part of the comment might be better placed above the config code since this is an implementation detail. Usually function level comment just describe the high level functionality so people know how to call it and what to expect.